File: sdkwork-backend-sdk-typescript/src/types/plus-agent-skill-form.ts
/** æè½åå»º/æ´æ°è¡¨å */
export interface PlusAgentSkillForm {
  /** æè½å½å±ç¨æ·IDï¼ä¸ä¼ é»è®¤å½åç¨æ· */
  userId?: number;
  /** æè½å¯ä¸æ è¯ */
  skillKey?: string;
  /** æè½åç§° */
  name?: string;
  /** æè½ç®ä» */
  summary?: string;
  /** æè½æè¿° */
  description?: string;
  /** æè½å¾æ  */
  icon?: string;
  /** æè½å°é¢å¾ */
  coverImage?: string;
  /** åç±»ID */
  categoryId?: number;
  /** ååID */
  packageId?: number;
  /** æä¾å */
  provider?: string;
  /** çæ¬ */
  version?: string;
  /** è¿è¡æ¶ */
  runtime?: string;
  /** å¥å£ */
  entrypoint?: string;
  /** manifestå°å */
  manifestUrl?: string;
  /** ä»£ç ä»åºå°å */
  repositoryUrl?: string;
  /** ä¸»é¡µå°å */
  homepageUrl?: string;
  /** ææ¡£å°å */
  documentationUrl?: string;
  /** è®¸å¯è¯åç§° */
  licenseName?: string;
  /** æ¥æºç±»å */
  sourceType?: 'OFFICIAL' | 'COMMUNITY' | 'ENTERPRISE' | 'PRIVATE' | 'CUSTOM';
  /** å¯è§æ§ */
  visibility?: 'PUBLIC' | 'PRIVATE' | 'UNLISTED';
  /** æ¯å¦å¯ç¨ */
  enabled?: boolean;
  /** ä»·æ ¼ */
  price?: number;
  /** è´§å¸ */
  currency?: string;
  /** æ ç­¾ */
  tags?: string[];
  /** è½ååè¡¨ */
  capabilities?: string[];
  /** éç½®Schema */
  configSchema?: Record<string, unknown>;
  /** é»è®¤éç½® */
  defaultConfig?: Record<string, unknown>;
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_agent_skill_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusAgentSkillForm:
    """技能创建/更新表单"""
    user_id: int = None
    skill_key: str = None
    name: str = None
    summary: str = None
    description: str = None
    icon: str = None
    cover_image: str = None
    category_id: int = None
    package_id: int = None
    provider: str = None
    version: str = None
    runtime: str = None
    entrypoint: str = None
    manifest_url: str = None
    repository_url: str = None
    homepage_url: str = None
    documentation_url: str = None
    license_name: str = None
    source_type: str = None
    visibility: str = None
    enabled: bool = None
    price: float = None
    currency: str = None
    tags: List[str] = None
    capabilities: List[str] = None
    config_schema: Dict[str, Any] = None
    default_config: Dict[str, Any] = None
